Simulator/Simulator.WebJob/app.config

```diff
@@ -64,6 +82,18 @@
           </ArrayOfString>
         </value>
       </setting>
-    </Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WorkerRole.Properties.Settings>
+    </Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WebJob.Properties.Settings>
   </applicationSettings>
-</configuration>
+  <system.web>
+    <membership defaultProvider="ClientAuthenticationMembershipProvider">
```

Comment:
Could you please explain why we need the `ClientAuthenticationMembershipProvider` here? My understanding is that it is related to forms authentication--does the simulator use forms auth?